build-parent/pom.xml

```diff
@@ -454,6 +454,12 @@
                     <artifactId>maven-resources-plugin</artifactId>
                     <version>3.1.0</version>
                 </plugin>
+                <plugin>
```

Comment:
Was this needed for some reason?

Reply:
Yes, so the way the immutable object recording works is that it maps field names to constructor parameter names. To actually record these names you need to use the -parameters javac option. Otherwise you would need annotations to specify the parameter names.

Reply:
:+1: 

I was just wondering about the placement of this configuration (I did the review in the IDE and it seemed to not have updated the comment), not the functionality itself.